packages/base/room.gts

```diff
@@ -579,11 +579,6 @@ export class RoomField extends FieldDef {
         } else if (event.content.msgtype === 'org.boxel.command') {
           // We only handle patches for now
           let command = event.content.data.command;
-          if (command.type !== 'patch') {
-            throw new Error(
-              `cannot handle commands in room with type ${command.type}`,
-            );
-          }
```

Comment:
We just let all commands thru 





![Screenshot 2024-05-14 at 20 40 49](https://github.com/cardstack/boxel/assets/8165111/da8d283f-033a-4411-b3b2-4943674cee37)

Reply:
@backspace ur right. There will be 'patch' event and 'patchCard' staying in the server. Previously, we threw an error in the card if it was not 'patch', but I just let both commands go thru now. 

The old message will display in the UI with no apply buttons. Tne new message will display in the UI with an apply button. I think this is backward compatible. 

The risk is if there was another commandType which I didn't consider apart from 'patch' and 'patchCard'